liquidhaskell.cabal

```diff
@@ -271,4 +274,6 @@ test-suite test
                      tasty >= 0.10,
                      tasty-hunit >= 0.9,
                      tasty-rerun >= 1.1,
-                     transformers
+                     transformers,
+                     Cabal >= 1.18
```

Comment:
We _do_ need it to build Liquid.hs --> CmdLine.hs --> Cabal.hs

That is, we need it in the executable too.

I'll let you fiddle with the Cabal file, I can't get it to build without
putting the damn thing everywhere :)

have fixed the putStrLn issues...
​